docs/content/docs/test/TestDataClearing.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Test Data Clearing
```

Comment:
What is the use case for the user? Why does he need this explanation?

Reply:
It was not obvious for me and I've thought that it could be not obvious for somebody else.

Reply:
And in the future it can be helpful for solving a problem when one test sees data left from another one.

Reply:
Remove this document completely